doc/provides.rst

```diff
@@ -167,5 +167,14 @@ Campaigns Provide Categories
 ``campaign_context_condition``
     Context Conditions that matches against the current context in shop to see if campaign matches.
 
+``catalog_campaign_effect``
+    Effect for matching ``campaign_catalog_filter`` or/and ``campaign_context_condition``
```

Comment:
* I don't understand the word "matching" here. Neither how filter or condition have anything to do with the effect.
* The other provides are named by prefixing them with the `catalog_` IMHO this should be too.
* It's actually a form, not effect.

How about this? (the provide key would have to be changed too)

```rst
``campaign_product_discount_effect_form``
   Form for handling product discount effects of a catalog campaign.
   Should be a ModelForm with its model being a subclass of
   `~shoop.campaigns.models.ProductDiscountEffect`.
```